components/suggestion/command_provider/command_provider.tsx

```diff
@@ -139,13 +141,29 @@ export default class CommandProvider extends Provider {
         const channel = this.isInRHS && selectedPost.channel_id ? getChannel(store.getState(), selectedPost.channel_id) : getCurrentChannel(store.getState());
 
         const args = {
-            channel_id: channel?.id,
+            channel_id: channel.id,
             ...(rootId && {root_id: rootId, parent_id: rootId}),
         };
 
-        Client4.getCommandAutocompleteSuggestionsList(command, teamId, args).then(
+        if (shouldHandleCommand(command, getCloudAppCommandLocations(store.getState()))) {
+            handleCommand(command, getCloudAppCommandLocations(store.getState()), args).then((matches => {
+                const terms = matches.map((suggestion) => suggestion.complete);
+                resultCallback({
+                    matchedPretext: command,
+                    terms,
+                    items: matches,
+                    component: CommandSuggestion,
+                });
+            }))
+            return;
+        }
+
+        const syncSuggestions = getSynchronousResults(pretext, getCloudAppCommandLocations(store.getState()));
```

Comment:
Also, we are trying to move away from the "Cloud App" concept and leave it just as "App".